README.md

```diff
@@ -1,33 +1,123 @@
 
-The goal is to refine the metrics that inform [Growth-Maturity-Decline][gmd] and to work with software implementations.
+This is a [CHAOSS](https://chaoss.community) working group, 
+focused on the [Growth-Maturity-Decline][gmd] metrics category.
+Its aim is to coordinate the definition of the metrics, and
+the production of software for implementing them, in this area.
 
-[gmd]: https://github.com/chaoss/metrics/blob/master/2_Growth-Maturity-Decline.md
+[gmd]: gmd_metrics.md
 
-## Meetings
-Every other Wednesday at 11am CDT / 18:00 CET. ([local time](http://www.thetimezoneconverter.com/?t=11am&tz=Chicago&)) in the CHAOSS Zoom room https://unomaha.zoom.us/j/720431288 (Meeting ID for [dial in](https://unomaha.zoom.us/zoomconference?m=DKGo2mmIuOv9xSjphoGZZmYKxr5HFrS9): 720 431 288)
+# How to participate
+
+You can start by introducing yourself on the CHAOSS mailing list (see below) explaining your interest.
+Then, you can have a look at the archives of the mailing lists, at the minutes of past meetings,
+and at the [issues](https://github.com/chaoss/wg-gmd/issues) and
+[pull requests](https://github.com/chaoss/wg-gmd/pulls) in
+[this repository](https://github.com/chaoss/wg-gmd).
+
+Then, of course you can participate in the mailing list,
+in online meetings, and in issues and pull requests.
+
+Currently, main lines of work are:
+
+* [Use cases](use_cases). Proposal and discussion of use cases that help to understand metrics in context.
+You can propose your use cases, and/or contribute to refine those that have been proposed already.
+
+* Focus areas (definitions, goals, questions). Currently, in the [definition of GMD metrics] we strucuture
+it in the following focus areas: [Issue Resolution](focus_areas/issue_resolution.md), [Code Development](focus_areas/code_development.md), and [Community Growth](focus_areas/communitiy_growth.md).
+For each area of interest, we're following the [goal-question-metric methodology](https://en.wikipedia.org/wiki/GQM) defining questions and metrics that help to answer them.
+You can contribute by proposing new goals for a focus area, or new questions for learning about those goals,
+or new metrics for answering those questions. Or by helping to refine goals, and questions
+(see about metrics below).
+
+* Metrics definition. For each metric, we have a document describing it,
+all of them in the [metrics directory](metrics).
+You can contribute by helping to refine those metrics definitions.
+
+* Reference implementations. For each metric, we intend to produce reference implementations,
+in the [implementations directory](implementations).
+They are based on data from real data sources,
+retrieved using [Perceval](https://github.com/chaoss/grimirelab-perceval).
+The idea is to first use a Python notebook to study how to produce the metric, and
+all the variations, parameters, etc, that are convenient to have into account.
+And then, produce a simple script that will compute the metric from a Perceval dump.
```

Comment:
From a "data source".  I think the examples can and should describe a variety of data provenance strategies.  We have GHTorrent, GitArchive, Facade, Git Repos themselves and the GitHub API as data sources independent of other collection tools like Perceval.  I think we want to keep the instructions and examples more general.

Reply:
(i.e., the source data structure is one factor. ... but it could be abstracted in the examples)